### ISSUE TITLE
poor cifar100 results

### ISSUE THREAD

**Issue by jcpeterson** [titu1994/DenseNet#39]
The standard cifar100 model only obtains about 60% validation accuracy when letting the standard script run for as many epochs as it wants before stopping early.

**Comment by titu1994** [comment]
Interesting. I have tried Cifar 100 with this model a long time ago and it did get into the 70s at least. I didn't use early stopping though, and ran for 300 epochs, with learning rate halved when there was no improvement after 20 epochs.
